### PR TITLE
CELDEV-1184 - remove/reduce old velocity macros

### PR DESCRIPTION
**Pull request by fpichler**
https://synjira.atlassian.net/browse/CELDEV-1184

### CODE REVIEW

File: src/main/webapp/templates/editactions.vm [hunk 1]
```diff
@@ -3,7 +3,7 @@
 #if ("$!showactions" != '0')
   $xwiki.jsfx.use('js/xwiki/actionbuttons/actionButtons.js', true)
   $xwiki.ssfx.use('js/xwiki/actionbuttons/actionButtons.css', true)
-  #set ($keyboardShortcutsEnabled = ($editor != 'wysiwyg'))
+  #set ($keyboardShortcutsEnabled = false)
```
Comment (msladek) [inline src/main/webapp/templates/editactions.vm]: should always be `true` now without `wysiwyg`?
```suggestion
  #set ($keyboardShortcutsEnabled = true)
```

File: src/main/webapp/skins/toucan/editmodes.vm
Comment (msladek) [inline src/main/webapp/skins/toucan/editmodes.vm:1]: we use the toucan skin in e.g. the main wikis. could removing it cause issues there?

**Review by msladek — CHANGES_REQUESTED** [review]

File: src/main/webapp/skins/toucan/editmodes.vm
Comment (fpichler) [reply src/main/webapp/skins/toucan/editmodes.vm:1]: albatross is the base skin and will be used instead. Thus nope.

File: src/main/webapp/templates/editactions.vm (hunk 1 above)
Comment (fpichler) [reply src/main/webapp/templates/editactions.vm]: according to jira issue, the support should for shortcuts is completly removed. Thus I removed it. :-)

**Review by msladek — APPROVED** [review]

**Pull request merged**